mrchecker-framework-modules/mrchecker-webapi-module/src/test/java/com/capgemini/mrchecker/webapi/httpbin/HtmlTest.java

```diff
@@ -0,0 +1,45 @@
+package com.capgemini.mrchecker.webapi.httpbin;
+
+import com.capgemini.mrchecker.test.core.logger.BFLogger;
+import com.capgemini.mrchecker.webapi.BasePageWebApiTest;
+import com.capgemini.mrchecker.webapi.pages.httbin.HtmlPage;
+import io.restassured.response.Response;
+import io.restassured.response.ResponseBody;
+import org.jsoup.Jsoup;
+import org.jsoup.nodes.Document;
+import org.jsoup.select.Elements;
+import org.junit.Test;
+
+import static org.hamcrest.Matchers.is;
+import static org.junit.Assert.assertThat;
+
+public class HtmlTest extends BasePageWebApiTest {
+	private HtmlPage htmlPage = new HtmlPage();
+
+	@Test
+	public void validateCorrectnessOfHtmlPage() {
+		int h1ElementsCount = 1;
+		String h1ElementText = "Herman Melville - Moby-Dick";
+		int pElementCount = 1;
+		int pElementTextLength = 3566;
+
+		BFLogger.logInfo("Step 1 - Sending GET query to " + htmlPage.getEndpoint());
+		Response response = htmlPage.getHtmlDocument();
+
+		BFLogger.logInfo("Step 2 - Validate response status code: ");
+		assertThat(response.statusCode(), is(200));
+
+		BFLogger.logInfo("Step 3 - Validate response body is html that contains proper h1 tag content (count and value)");
+		ResponseBody body = response.body();
```

Comment:
Duplication with https://github.com/devonfw-forge/mrchecker-source/pull/12/files
Please consider to create some class to wrap HTML functionality

Reply:
Please tell me more specific where is this duplication. Here is get request that just return some body with html page and in linked case there is get request with params that returns body with html page with links dependand of those params. For both cases I use Jsoup to parse and check content of pages. Maybe I missing something but I can't see what should be moved to some class for HTML. This should be some wrapper for Jsoup?

Reply:
I guess Pawel means to create some helper class that would deal with HTML parsing (with Jsoup), so it's not duplicated in multiple test classes.

Reply:
That's right. Like Tomek said. It will be helpful with parsing HTML pages for future tests created using this module.